lib/schemas/FieldSchema.js

```diff
@@ -101,6 +101,10 @@ module.exports = makeSchema({
       description: 'Does the value of this field affect the definitions of other fields in the set?',
       type: 'boolean',
     },
+    inputFormat: {
```

Comment:
I'm wondering if we should remove the `,` from the `inputFormat` value (because while it simplifies for the backend, there's no real justification for it in this "public-facing" place. It's arguably cleaner if it was just `https://{{subdomain}}.yourdomain.com/`, and since it applies to subdomain only, should this field name be `subdomainFormat` instead of `inputFormat`?)

Reply:
I feel that `inputFormat` could do more than just subdomain, so I've changed the description to be more general.

Reply:
Makes sense!